pallets/creditcoin/src/lib.rs

```diff
@@ -62,10 +59,11 @@ pub mod pallet {
 	use super::*;
 	use frame_support::{
 		dispatch::DispatchResult,
-		pallet_prelude::*,
+		pallet_prelude::{StorageDoubleMap, *},
```

Comment:
`StorageDoubleMap` is redundant because of the glob import

Reply:
It is. But it helps me greatly when I'm not using linters, too many glob imports hinders readability if you are not that familiar with the code imo. Of course I can change it if it is a big deal.